lib/tasks/eu_exit_business_finder.rake

```diff
@@ -0,0 +1,102 @@
+namespace :eu_exit_business_finder do
+  task update_subscriber_list_titles_and_facet_values: [:environment] do
+    subscriber_lists = SubscriberList
+      .where("links::text != '{}'")
+      .where("ARRAY(SELECT json_object_keys(links)) <@ Array[:keys]", keys: [:facet_values])
+    update_facet_values(subscriber_lists)
+  end
+
+  # Iterate through the affected subscriber lists and if we encounter any
+  # facet values that need replacing with a newer one, we insert the new one
+  # to the array (inserted 'in place' to retain original order). Then remove
+  # the old facet values in bulk and write only the unique values that remain.
+  def update_facet_values(subscriber_lists)
+    subscriber_lists.each do |list|
+      links = list.links[:facet_values][:any]
+      updated_links = links.map { |original_val| replace_facet_val(original_val) || original_val }
+      updated_links = (updated_links - EuExitFacetMigrationConfig::facet_values_to_remove).uniq
+      list.write_attribute('links', facet_values: { any: updated_links })
+      list.write_attribute('title', title_from_facets(updated_links))
+      list.save
+    end
+  end
+
+  def replace_facet_val(original_facet_val)
```

Comment:
Could you rename `val` to `value` in `replace_facet_val` and `replace_val`, just so it's clear what this value is?